tests/storage-spec.js

```diff
@@ -131,6 +131,12 @@ describe('languageStorage factory', function () {
         });
 
         it('should set the default language from the cookie', function () {
+            // This test won't pass in angular < 1.4 due to how $browser (a private service)
+            //  is mocked so it doesn't read actual cookies from the document
+            const angularVersion = window.__karma__.config.args[0];
+            if (angularVersion === '1.2.15' || angularVersion === '1.3') {
```

Comment:
Can you `parseFloat` and compare `< 1.4` instead of statically checking version numbers? You can check just the first three characters:
```js
parseFloat(angularVersion.slice(0, 2)) < 1.4
```